.gitlab/packaging/deb.yml

```diff
@@ -166,14 +166,19 @@ iot_agent_deb-arm64:
 
 iot_agent_deb-armhf:
   extends: .package_iot_deb_common
-  image: registry.ddbuild.io/ci/datadog-agent-buildimages/deb_armhf$CI_IMAGE_DEB_ARMHF_SUFFIX:$CI_IMAGE_DEB_ARMHF
+  image: registry.ddbuild.io/ci/datadog-agent-buildimages/linux$CI_IMAGE_LINUX_SUFFIX:$CI_IMAGE_LINUX
   tags: ["arch:arm64"]
   needs: ["iot-agent-armhf"]
   variables:
     PACKAGE_ARCH: armhf
     DD_PKG_ARCH: "arm64"
     FORCED_PACKAGE_COMPRESSION_LEVEL: 5
     STATIC_QUALITY_GATE_NAME: "static_quality_gate_iot_agent_deb_armhf"
+  script:
+    - !reference [.cache_omnibus_ruby_deps, setup]
+    - !reference [.setup_deb_signing_key]
+    - PATH="/opt/dpkg-armhf/bin:$PATH" dda inv -- -e omnibus.build --base-dir $OMNIBUS_BASE_DIR --skip-deps --flavor iot
```

Comment:
```suggestion
    # Use custom built dpkg for armhf packaging since we're on armv8/arm64
    - PATH="/opt/dpkg-armhf/bin:$PATH" dda inv -- -e omnibus.build --base-dir $OMNIBUS_BASE_DIR --skip-deps --flavor iot
```